src/mca/common/dstore/dstore_file.h

```diff
@@ -0,0 +1,161 @@
+#ifndef DSTORE_FORMAT_H
+#define DSTORE_FORMAT_H
+
+typedef size_t (*pmix_common_dstore_kv_size_fn)(uint8_t *addr);
+typedef char* (*pmix_common_dstore_key_name_ptr_fn)(uint8_t *addr);
+typedef size_t (*pmix_common_dstore_key_name_len_fn)(char *key);
+typedef uint8_t* (*pmix_common_dstore_data_ptr_fn)(uint8_t *addr);
+typedef size_t (*pmix_common_dstore_data_size_fn)(uint8_t *addr, uint8_t* data_ptr);
+typedef size_t (*pmix_common_dstore_key_size_fn)(char *key, size_t data_size);
+typedef size_t (*pmix_common_dstore_slot_size_fn)(void);
```

Comment:
Is this for the extension slot? If so this should be reflected in the name of a function

Reply:
Updated.